tournament.screenlayout.inc

```diff
@@ -1,3 +1,6 @@
+.define INGAME_NAM "build/tournament.nam.rle"
```

Comment:
I noticed the initial drawing of the playfield was weird, because I could see the curtain being cleared from the playfield. It appears `tournament.nam` has the curtain in the playfield, instead of empty/black ($FF or $EF). You probably want to remove the curtains from the nametable to avoid that visual glitch.

If you are aware of this and using it as a debug tool or something, it is fine to leave. I just wanted to mention it in case you hadn't realize it.

Reply:
The nametable contains quite a bit of unused stuff and needs some cleaning. I however wanted to investigate first, what should be on the nametable to have as few updates as possible in the first frames of the game because changing all number will take quite some frames with my new code.